tikv-configuration-file.md

```diff
@@ -1815,3 +1815,13 @@ To reduce write latency and avoid frequent access to PD, TiKV periodically fetch
 + TiKV adjusts the number of cached timestamps according to the timestamp consumption in the previous period. If the usage of locally cached timestamps is low, TiKV gradually reduces the number of cached timestamps until it reaches `renew-batch-min-size`. If large bursty write traffic often occurs in your application, you can set this parameter to a larger value as appropriate. Note that this parameter is the cache size for a single tikv-server. If you set the parameter to too large a value and the cluster contains many tikv-servers, the TSO consumption will be too fast.
 + In the **TiKV-RAW** \> **Causal timestamp** panel in Grafana, **TSO batch size** is the number of locally cached timestamps that has been dynamically adjusted according to the application workload. You can refer to this metric to adjust `renew-batch-min-size`.
 + Default value: `100`
+
+### `s3-multi-part-size` <span class="version-mark">New in v5.3.2</span>
+
+> **Note:**
+>
+> This configuration is introduced to address backup failures caused by S3 rate limiting. This problem has been fixed by [refining the backup data storage structure](/br/backup-and-restore-design.md#backup-file-structure) in v6.1.1, v6.2.0, and later versions. Therefore, this configuration is now deprecated and is no longer recommended.
```

Comment:
I think simply saying that this config is deprecated is enough for v6.3 and dev users.

Reply:
@IANTHEREAL I think it makes sense. What do you think?

Reply:
just describe the fact

Reply:
Updated the doc in [ef08e0e](https://github.com/pingcap/docs/pull/10316/commits/ef08e0eed15d6d7d3e7c72978f7a724cafd41473). PTAL @ran-huang